NOTICE

```diff
@@ -1,5 +1,6 @@
-Copyright 2020 My.com B.V. (Mail.Ru Group)
-Copyright 2021-2025 Organic Maps Contributors
+Copyright 2010-2021 My.com B.V. (Mail.Ru Group)
```

Comment:
2010-2021 are actual dates based on git history. 

MAPS.ME team continued to update the upstream repository until the [final commit](https://github.com/mapsme/omim/commit/1892903b63f2c85b16ed4966d21fe76aba06b9ba) made on *April 28, 2021*. Commits made by the Mail.Ru team to the mapsme/omim repository after the fork date were later rebased and merged into the this repository.

Reply:
Updated.